### DISCUSSION TITLE
Is it possible to load site from /dist folder?

### DISCUSSION THREAD

**Discussion by lostsatellites** [community/community#22324] [Other Feature Feedback, Questions, & Ideas]
<p>My setup includes a /source and /dist directory and I’m wondering if it’s possible to serve up the site from /dist? This is not a Jekyll site, I’m only using Gulp tasks to compile and move files from /source to /dist. Any help is appreciated.</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/lostsatellites">@lostsatellites</a>,</p>
<p>Thanks for being here! Two things for your reference <a href="https://github.com/shinnn/gulp-gh-pages" rel="nofollow noopener">Gulp</a> has a way to do this directly or you can create a bash script ie:</p>
<pre><code>git subtree push --prefix dist origin gh-pages
</code></pre>
<p>Otherwise, you can deploy your pages from a doc folder as seen in the <a href="https://help.github.com/articles/configuring-a-publishing-source-for-github-pages/#publishing-your-github-pages-site-from-a-docs-folder-on-your-master-branch" rel="nofollow noopener">documentation</a>.</p>
<p>I hope this helps!</p>

**Comment by lostsatellites** [comment]
<p>Thanks <a class="mention" href="/andreagriffiths11">@andreagriffiths11</a>! A couple of good options to explore here.</p>
